e2e/agent/agent_e2e_test.go

```diff
@@ -242,11 +250,40 @@ func Test_Agent(t *testing.T) {
 					return
 				}
 
-				err = tc.check.Verify(k8s, cluster.Ingress(), namespace)
-				if err != nil {
-					t.Errorf("failed : %v", err)
-					return
-				}
+				t.Run("via ingress", func(t *testing.T) {
+					t.Parallel()
+
+					err := tc.check.Verify(k8s, cluster.Ingress(), namespace)
+					if err != nil {
+						t.Errorf("failed : %v", err)
+						return
+					}
+				})
+
+				t.Run("via port-forward", func(t *testing.T) {
```

Comment:
I think we are repeating here the same problem as when the tests were in the disruptor e2e test, putting them as a sub-test of the fault injection instead of an independent test. However, this is easy to change in a follow up PR so I will not ask to change this.

Reply:
My main goal was to reuse the setup, as it is largely identical, but I don't have a strong opinion on it. We can split-and-copypaste if that's preferred.